aten/src/ATen/native/BinaryOps.cpp

```diff
@@ -62,12 +62,11 @@ Tensor& add_(Tensor& self, const Tensor& other, Scalar alpha) {
 
 Tensor& div_out(Tensor& result, const Tensor& self, const Tensor& other) {
   if (isIntegralType(result.scalar_type(), /*includeBool=*/ true)) {
-    TORCH_WARN_ONCE(
-      "Integer division of tensors using div or / is deprecated, ",
-      "and in a future release div will perform true division as in Python 3. ",
+    TORCH_CHECK(false,
+      "Integer division of tensors using div or / is no longer supported, ",
+      "and in a future release div will perform true division like Python 3. ",
```

Comment:
nit: why'd you change as in? :).

Reply:
Which do you prefer?

Reply:
From review: revert to "as in" (because "like" is too SoCal)